packages/core/src/Stores/Helpers/chart-marker-helpers.js

```diff
@@ -104,22 +108,26 @@ export const createMarkerSpotExit = (contract_info, tick, idx) => {
               align_label: should_show_profit_label ? 'middle' : align_label,
               spot_count,
               spot_profit:
-                  should_show_profit_label &&
-                  `${formatMoney(contract_info.currency, contract_info.profit, true)} ${contract_info.currency}`,
+                  (should_show_profit_label &&
+                      `${formatMoney(contract_info.currency, contract_info.profit, true)} ${contract_info.currency}`) ||
+                  '',
```

Comment:
fixing this error:
<img width="343" alt="Screenshot 2023-03-19 at 2 25 12 PM" src="https://user-images.githubusercontent.com/103177211/226172153-c3fd1891-06f6-400b-bf67-e958e5327820.png">